src/types.ts

```diff
@@ -332,6 +332,11 @@ interface ReleaseRcOptions {
    * Which tag to publish the rc as
    */
   tag: string
+
+  /**
+   * Don't prefix output with the package name
+   */
+  noPrefix?: boolean
```

Comment:
In lieu of tests, when trying this out locally this CLI argument name ends up a bit weird:

```console
% npx aegir exec --no-prefix ls -la
// data is prefixed
```

```console
% npx aegir exec --no-prefix=true ls -la
// data is not prefixed
```

Maybe having the cli arg as `prefix` would be better?:

```console
% npx aegir exec ls -la
// data is prefixed
```

```console
% npx aegir exec --prefix=false ls -la
// data is not prefixed
```

Then we get the following for free because yargs:

```console
% npx aegir exec --no-prefix ls -la
// data is not prefixed
```